src/main/kotlin/no/skatteetaten/aurora/gobo/resolvers/applicationdeployment/ApplicationDeploymentResolver.kt

```diff
@@ -23,7 +23,7 @@ class ApplicationDeploymentResolver(
         Namespace(applicationDeployment.namespaceId, applicationDeployment.affiliationId)
 
     fun details(applicationDeployment: ApplicationDeployment, dfe: DataFetchingEnvironment) =
-        dfe.loader(ApplicationDeploymentDetails::class).load(applicationDeployment.id)
+        dfe.loader<ApplicationDeploymentDetails>().load(applicationDeployment.id)
```

Comment:
Jeg må innrømme at jeg egentlig ikke syntes dette er bedre. Det er ikke alltid det passer å bruke reification og generics for å overføre typeinformasjon. Etter min vurdering leser

dfe.loaderFor(ApplicationDeploymentDetails::class)

bedre enn

dfe.loader<ApplicationDeploymentDetails>() eller evt. dfe.loaderFor<ApplicationDeploymentDetails>()

Å bruke generics her er bare en konvoluted måte å sende inn en parameter på. En måte å vurdere om generics passer eller ei på er å sjekke om metodesignaturen passer ved å gå over til flertall. F.eks.

val loaders = dfe.loadersFor(ApplicationDeploymentDetails::class, DeploymentSpec::class)

En slik endring vil ikke kunne støttes med bruk av generics.